docker-entrypoint.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Migrations
+python emission_events/manage.py migrate
+
+# Collect static files
+python emission_events/manage.py collectstatic --noinput
+
+# Run app
+cd emission_events
```

Comment:
Oh, I see what's happening. I'd tack this on to the gunicorn call instead.

`emission_events.emission_events.wsgi:application`